### DISCUSSION TITLE
googleAnalytics config issues

### DISCUSSION THREAD

**Discussion by anthony83** [vaga/hugo-theme-m10c#71] [Q&A] [answered]
Hi There,

Ive created an analytics account however after providing my trackingID against parameter googleAnalytics within the config.toml file i do not appear to get any results in google analytics. Are there other areas or files to be considered when configuring google analytics with this template?

**Comment by vaga** [comment, marked as answer]
You should put `googleAnalytics` propertry at the same level of `theme` property. (not in `params`)

```
...
theme = "m10c"
googleAnalytics = "......"
...
```

**Reply by anthony83** [reply]
thanks, its definitely outside the `params` and im still not able to make the connection.

`baseurl = "/"
title = "title"
author = "You"
canonifyurls = true
paginate = 8

theme = "m10c"
googleAnalytics = "UA-xxxxxxx-x"

[menu]
  [[menu.main]]
    identifier = "home"
    name = "Home"
    url = "/"
    weight = 1`

**Reply by vaga** [reply]
In the generated HTML, do you have the GA script block?

**Reply by anthony83** [reply]
yes i can see it in the generated index.html file
`ga('create', 'UA-xxxxxxxxx-1', 'auto');
`

`ga('send', 'pageview');`

**Reply by vaga** [reply]
It should work... 😟  The theme includes the hugo internal template for Google Analytics.
https://github.com/gohugoio/hugo/blob/ba16a14c6e884e309380610331aff78213f84751/tpl/tplimpl/embedded/templates/google_analytics.html

**Reply by vaga** [reply]
If you use Firefox, the google analytics script is automatically disabled. Firefox has a Tracking Protection by default.

**Reply by anthony83** [reply]
i haven't been able to locate the file within the template:

https://github.com/gohugoio/hugo/blob/ba16a14c6e884e309380610331aff78213f84751/tpl/tplimpl/embedded/templates/google_analytics.html

the template references ` {{ template "_internal/google_analytics.html" . }}` but where is the file?

**Reply by anthony83** [reply]
Hi @vaga its working now, it was Firefox. 
Please close this off. The Integration works fine.
